translation/it/README.md

```diff
@@ -0,0 +1,90 @@
+![AI Generativa per Principianti](../../img/1.png)
+
+### Un corso di 12 lezioni che ti insegnerà tutto quello che devi sapere per iniziare a costruire applicazioni con l'AI Generativa
+
+[![GitHub license](https://img.shields.io/github/license/microsoft/Generative-AI-For-Beginners.svg)](https://github.com/microsoft/Generative-AI-For-Beginners/blob/master/LICENSE?WT.mc_id=academic-105485-koreyst)
+[![GitHub contributors](https://img.shields.io/github/contributors/microsoft/Generative-AI-For-Beginners.svg)](https://GitHub.com/microsoft/Generative-AI-For-Beginners/graphs/contributors/?WT.mc_id=academic-105485-koreyst)
+[![GitHub issues](https://img.shields.io/github/issues/microsoft/Generative-AI-For-Beginners.svg)](https://GitHub.com/microsoft/Generative-AI-For-Beginners/issues/?WT.mc_id=academic-105485-koreyst)
+[![GitHub pull-requests](https://img.shields.io/github/issues-pr/microsoft/Generative-AI-For-Beginners.svg)](https://GitHub.com/microsoft/Generative-AI-For-Beginners/pulls/?WT.mc_id=academic-105485-koreyst)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+
+[![GitHub watchers](https://img.shields.io/github/watchers/microsoft/Generative-AI-For-Beginners.svg?style=social&label=Watch)](https://GitHub.com/microsoft/Generative-AI-For-Beginners/watchers/?WT.mc_id=academic-105485-koreyst)
+[![GitHub forks](https://img.shields.io/github/forks/microsoft/Generative-AI-For-Beginners.svg?style=social&label=Fork)](https://GitHub.com/microsoft/Generative-AI-For-Beginners/network/?WT.mc_id=academic-105485-koreyst)
+[![GitHub stars](https://img.shields.io/github/stars/microsoft/Generative-AI-For-Beginners.svg?style=social&label=Star)](https://GitHub.com/microsoft/Generative-AI-For-Beginners/stargazers/?WT.mc_id=academic-105485-koreyst)
+
+[![Open in GitHub Codespaces](https://img.shields.io/static/v1?style=for-the-badge&label=GitHub+Codespaces&message=Open&color=lightgrey&logo=github)](https://codespaces.new/microsoft/generative-ai-for-beginners?WT.mc_id=academic-105485-koreyst)
+[![](https://dcbadge.vercel.app/api/server/ByRwuEEgH4)](https://aka.ms/genai-discord)
+
+# AI Generativa per Principianti
+
+Impara i fondamenti della creazione di applicazioni dell'AI Generativa con il nostro corso completo di 12 lezioni tenuto da Microsoft Cloud Advocates. Ogni lezione copre un aspetto fondamentale dei principi dell'AI Generativa e dello sviluppo app. Durante questo corso, costruiremo la nostra startup di AI Generativa in modo che tu possa capire quello che ci vuole per lanciare le tue idee.
```

Comment:
La riformulerei così: 'Impara i fondamenti della creazione di applicazioni di IA generativa con il nostro corso completo di 12 lezioni tenuto da Microsoft Cloud Advocates. Ogni lezione tratta un aspetto fondamentale dei principi dell'IA generativa e dello sviluppo applicativo. Durante questo corso, daremo vita alla nostra startup di IA generativa in modo che tu possa comprendere ciò che è necessario per realizzare le tue idee.'